coderd/database/queries/insights.sql

```diff
@@ -771,3 +771,66 @@ SELECT
 FROM unique_template_params utp
 JOIN workspace_build_parameters wbp ON (utp.workspace_build_ids @> ARRAY[wbp.workspace_build_id] AND utp.name = wbp.name)
 GROUP BY utp.num, utp.template_ids, utp.name, utp.type, utp.display_name, utp.description, utp.options, wbp.value;
+
+-- name: GetUserStatusCountsByDay :many
```

Comment:
@mafredri I'm still working on CI for this PR. The rest is not ready for review yet, but would you mind reviewing this sql query in the mean time please?